<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasks_adding</class>
 <widget class="QDialog" name="tasks_adding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PersoLife - добавление задач</string>
  </property>
  <property name="windowIcon">
   <iconset resource="meres.qrc">
    <normaloff>:/resours/resourses/icons/icons8-запись-48.png</normaloff>:/resours/resourses/icons/icons8-запись-48.png</iconset>
  </property>
  <property name="toolTip">
   <string>Окно добавления задач</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Временные промежутки задачи</string>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Выбор даты задачи</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit{
background: none;
border: 1px solid #000000;
border-radius: 5px;
}</string>
    </property>
    <property name="minimumDate">
     <date>
      <year>2023</year>
      <month>7</month>
      <day>11</day>
     </date>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_start">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>65</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выбор времени начала задачи</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTimeEdit{
background: none;
border: 1px solid #000000;
border-radius: 5px;
}</string>
    </property>
    <property name="time">
     <time>
      <hour>10</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>132</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время начала задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время конца задачи</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_end">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выбор времени конца задачи</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTimeEdit{
background: none;
border: 1px solid #000000;
border-radius: 5px;
}</string>
    </property>
    <property name="time">
     <time>
      <hour>12</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>321</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title">
    <string>Информация о задаче</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Поле ввода названия задачи</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: none;
	background-color: rgb(255, 255, 255);
	border: 1px solid #000000;
	border-radius: 5px;
}
QLineEdit::disabled{
	background-color: rgb(240, 240, 240);
	border: 1px solid #cccccc
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Приоритет</string>
    </property>
   </widget>
   <widget class="QSlider" name="during_slider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Слайдер выбора приоритета задачи</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Заметка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="note_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>65</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Поле ввода заметки задачи</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: none;
	background-color: rgb(255, 255, 255);
	border: 1px solid #000000;
	border-radius: 5px;
}
QLineEdit::disabled{
	background-color: rgb(240, 240, 240);
	border: 1px solid #cccccc
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>124</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>124</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>124</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>124</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>124</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Кнопка добавления задачи</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
               background: none;
               border: none;
               background-color: qlineargradient(spread:pad, x1:0.966, y1:1, x2:0.392, y2:0.420455, stop:0 rgba(59, 105, 162, 255), stop:1 rgba(96, 177, 236, 255));
               border: 1px solid #000000;
               border-color: qlineargradient(spread:pad, x1:0.966, y1:1, x2:0.392, y2:0.420455, stop:0 rgba(59, 105, 162, 255), stop:1 rgba(96, 177, 236, 255));
               border-radius:15px;
color:#fff;
           }
QPushButton::hover{
background: none;
		border: none;
	background-color: qlineargradient(spread:pad, x1:0.364, y1:0.420455, x2:1, y2:0.955, stop:0 rgba(90, 164, 255, 255), stop:0.664773 rgba(23, 79, 208, 255));
	border: 1px solid #000000;
	border-color: qlineargradient(spread:pad, x1:0.364, y1:0.420455, x2:1, y2:0.955, stop:0 rgba(90, 164, 255, 255), stop:0.664773 rgba(23, 79, 208, 255));
	border-radius:15px;
           }</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="meres.qrc"/>
 </resources>
 <connections/>
</ui>
